lib/llvm/Makefile

```diff
@@ -30,7 +77,9 @@ endif
 # Some defaults
 LLVM_BUILD_ENGINE ?= Unix Makefiles
 LLVM_BUILD_TYPE ?= Release
-LLVM_LINKER ?= gold
+
+# Define LLVM_LINKER to pass LLVM_USE_LINKER to have llvm output -fuse-ld=${LLVM_LINKER}
+#LLVM_LINKER ?= gold
```

Comment:
this should be defined for Linux yes?

Reply:
Regarding using gold, it doesn't seem to provide any real benefit at this point. Initially I'd used it because it used less memory and under certain circumstances might be faster. In the end I don't think there was much difference in time and we still need to limit the `CPUs` in .circleci/config.yml` so as not to use to much memory when linking. So, for now, lets leave it as is.

Reply:
Gotcha